Formula/jsign.rb

```diff
@@ -0,0 +1,24 @@
+class Jsign < Formula
+  desc "Tool for signing Windows executable files, installers and scripts"
+  homepage "https://ebourg.github.io/jsign/"
+  url "https://github.com/ebourg/jsign.git", tag: "4.2", revision: "e44d2474f7773cbade092362cd4a6a63810f0c95"
+  license "Apache-2.0"
+
+  depends_on "maven" => :build
+  depends_on "openjdk@17"
```

Comment:
any issue tracking down the latest openjdk support?

Reply:
Yes, a build plugin (proguard) doesn't work with OpenJDK 20 yet, and some tests use removed APIs. But at runtime Jsign works with the latest JDK. To be safe I stick to the latest LTS release.

Reply:
can you add that as code comment, that would be helpful

Reply:
Done